frontend/src/components/pages/AdminDashboard/ViewCheckInPage.tsx

```diff
@@ -1,12 +1,12 @@
 import { Container } from "@chakra-ui/react";
 import React from "react";
 
-import ViewDonations from "./ViewDonations";
+import ViewCheckins from "./ViewDonationsAndCheckins";
 
 const ViewDonationsPage = () => {
```

Comment:
rename to `ViewCheckinsPage`? Also I dont understand how we are able to import this as `CheckInsPage` in `App.tsx`, dont they have to be same name??